demos/demo_keygen.py

```diff
@@ -96,6 +96,8 @@ def progress(arg=None):
 
     if options.newphrase:
         phrase = getattr(options, 'newphrase')
+        # Turn it into bytes.
+        phrase=phrase.encode("utf-8")
```

Comment:
Of course! I put it on its own line, so I could comment on the need for it.

Also considered making the change to `_write_private_key()` at the same time, but figured there might be reasons I didnt know about, so didnt.

Would you like me to make the change to `_write_private_key()` as part of this PR, as a separate PR, or something else?